### DISCUSSION TITLE
Setting the Song Meta Data for BT Source

### DISCUSSION THREAD

**Discussion by degnarraer** [pschatzmann/ESP32-A2DP#331] [Q&A] [answered]
Is it possible to set the song Title, Album, etc... for the current song playing from a BT Source?

For example, I have an ESP32 sourcing BT audio. I also want to set the title, album, etc, so it appears on the Bluetooth sink.

**Comment by pschatzmann** [comment, marked as answer]
I never tried, so it is not part of my API, but feel free to extend it: Here is the [link to the relevant documentation](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/bluetooth/esp_avrc.html).
